qt/widgets/common/src/WorkspacePresenter/WorkspaceTreeWidget.cpp

```diff
@@ -110,18 +110,28 @@ void WorkspaceTreeWidget::setupWidgetLayout() {
 
   auto *buttonLayout = new FlowLayout();
   m_loadButton = new QPushButton("Load");
+  m_loadButton->setToolTip("Load a file or live data");
   m_saveButton = new QPushButton("Save");
+  m_saveButton->setToolTip("Save the selected workspaces");
   m_deleteButton = new QPushButton("Delete");
+  m_deleteButton->setToolTip("Deletes the selected workspaces");
```

Comment:
To be consistent with each other, I think we should drop the `s`, so `Delete the selected workspaces` and `Delete all workspaces`.